src/components/govukComponents/InsetText.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function InsetText(props) {
+
+  return (
+    <div
+      id={props.id}
+      className={`govuk-inset-text${props.classes ? props.classes : ''}`}
```

Comment:
@mick-jones I think this is missing a space? If props.classes is specified it will be munged into the previous string

Reply:
Fixed